index.html

```diff
@@ -300,27 +300,33 @@ <h3>
             Declaring multiple icons
           </h3>
           <p>
-            In the following example, the developer has made the following
-            choices about the icons associated with the web application:
+            This section shows how to declare multiple icons using the
+            [=manifest/icons=] member to declare a set of icons for your web
```

Comment:
Would need to check second-person usage? 

```suggestion
            [=manifest/icons=] member to declare a set of icons for a web
```